### PR TITLE
Remove netlify tracking from homebrew and go.

### PR DESCRIPTION
**Pull request by paulmillr**

### CODE REVIEW

**Comment by FiloSottile** [comment]
No, thank you. I want to own my namespaces and don't want to be tied to GitHub for hosting.

Not sure what tracking you are talking about, Netlify is my hosting provider, and surely brew and git don't send cookies along with their HTTPS requests. Moreover, Netlify doesn't provide me with logs just like GitHub.

Finally, I think you didn't test this, either, because `brew tap FiloSottile/age` won't work as it will look for github.com/FiloSottile/homebrew-age.

**Comment by paulmillr** [comment]
not relying on github is good - netlify request id seemed a bit shady to me. if there's no tracking, great.

```
~ ❯ curl -I https://filippo.io/age
HTTP/2 302 
cache-control: public, max-age=0, must-revalidate
content-length: 0
date: Wed, 22 Apr 2020 16:52:21 GMT
location: https://github.com/FiloSottile/age
strict-transport-security: max-age=63072000; includeSubDomains; preload
age: 0
server: Netlify
x-nf-request-id: e142dcd0-901d-44b8-9f85-f64e6ae99c05-2093734
```

**Comment by FiloSottile** [comment]
It's a response header, I don't see how it can be tracking the client.

**Comment by paulmillr** [comment]
@FiloSottile 

They assign GUIDs to every request. And the GUID repeats itself if you make request from one machine. See:

```
~ ❯ curl -I https://filippo.io/age
x-nf-request-id: 23add3b0-3ec4-4725-a2d8-f3127c563d15-2187632
~ ❯ curl -I https://filippo.io/age
x-nf-request-id: 23add3b0-3ec4-4725-a2d8-f3127c563d15-2188034
~ ❯ curl -I https://filippo.io/age
x-nf-request-id: 23add3b0-3ec4-4725-a2d8-f3127c563d15-2188463
```

That seems like tracking to me.

**Comment by FiloSottile** [comment]
Sounds more like a serving machine ID to me. Again, I don't see the point of sending a tracking ID _to the client_ in a response header. Anyway, Netlify is where my website is hosted, and I want my software to be tied to my website, not to my GitHub.

**Comment by bdd** [comment]
Googling `"x-nf-request-id" netlify` gives you this as the first hit where a support engineer elaborates what this request id does and how it helps them troubleshoot issues.
https://community.netlify.com/t/common-issue-netlify-support-asked-for-the-x-nf-request-id-header-why-are-they-asking-for-it-and-how-do-i-find-it/4385

tl;dr—it's a [tracing](https://opentracing.io/docs/overview/what-is-tracing/) id so they can refer to their logs and navigate from edge to the very bottom of their serving stack handling a particular request.

_P.S. I also checked the same search with DDG, in case maybe you don't use Google for tracking sensitivity. It's also the first result there._
